admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -125,6 +125,7 @@ export default class OrderProductRenderer {
 
   paginate(numPage) {
     const $rows = $(OrderViewPageMap.productsTable).find('tr[id^="orderProduct_"]');
+    const $customizationRows = $(OrderViewPageMap.productsTable).find('.order-product-customization');
```

Comment:
@atomiix May be create a line in OrderViewPageMap directly for avoiding the find

Reply:
Done